### PR TITLE
CO-1975 Mass mailing donation view changed

### PR DESCRIPTION
**Pull request by quentingigon**

### CODE REVIEW

File: mass_mailing_switzerland/models/mass_mailing_campaign.py [hunk 1]
```diff
@@ -55,6 +57,18 @@ def open_unsub(self):
     def open_clicks(self):
         return self.mass_mailing_ids.open_clicks()
 
+    @api.model
+    def fields_view_get(self, view_id=None, view_type='form', toolbar=False,
```
Comment (ecino) [inline mass_mailing_switzerland/models/mass_mailing_campaign.py]: For what purpose did you add this function? I'm not sure it is needed.

Comment (quentingigon) [reply mass_mailing_switzerland/models/mass_mailing_campaign.py]: Old fix tentative that I forgot to remove, sorry !

File: mass_mailing_switzerland/models/mass_mailing_campaign.py [hunk 2]
```diff
@@ -12,6 +12,8 @@
 
 from odoo.addons.queue_job.job import job, related_action
 
+from lxml import etree
```
Comment (ecino) [inline mass_mailing_switzerland/models/mass_mailing_campaign.py]: Now this import is useless.

File: mass_mailing_switzerland/models/mass_mailing_campaign.py [hunk 3]
```diff
@@ -54,7 +56,7 @@ def open_unsub(self):
     @api.multi
     def open_clicks(self):
         return self.mass_mailing_ids.open_clicks()
-
+
```
Comment (ecino) [inline mass_mailing_switzerland/models/mass_mailing_campaign.py]: You added useless spaces

**Review by ecino — CHANGES_REQUESTED** [review]

**Comment by quentingigon** [comment]
I thought I pushed these..

**Review by ecino — APPROVED** [review]

**Pull request merged**